C-Plus-Plus/other/Phone_keypad.cpp

```diff
@@ -0,0 +1,64 @@
+
+#include <iostream>
+#include <string>
+using namespace std;
+
+void printKeypad_rec(int num, string output){
+    string index[8] = {"abc", "def", "ghi", "jkl", "mno", "pqrs", "tuv", "wxyz"};
+    if(num == 0){
+        cout << output <<endl;
+        return;
+    }
+    
+    int rem = num%10;
+    int first = num/10;
+    if(rem == 7 || rem == 9){
+        printKeypad_rec(first, index[rem - 2][0] + output);
+		printKeypad_rec(first, index[rem - 2][1] + output);
+        printKeypad_rec(first, index[rem - 2][2] + output);
+        printKeypad_rec(first, index[rem - 2][3] + output);
+    }
+    else{
+        printKeypad_rec(first, index[rem - 2][0] + output);
+		printKeypad_rec(first, index[rem - 2][1] + output);
+        printKeypad_rec(first, index[rem - 2][2] + output);
+    }
+}
+
+void printKeypad(int num){
+    /*
+    Given an integer number print all the possible combinations of the keypad. You do not need to return anything just print them.
+    */
+    
+    string output = "";
+    printKeypad_rec(num, output);
+}
+
+int main(){
+    int num;
+    cin >> num;
+
+    printKeypad(num);
```

Comment:
Display message in print form as what are you showing output to the user for better UI